dipy/direction/tests/test_peaks.py

```diff
@@ -531,31 +531,34 @@ def test_peaksFromModelParallel():
     data, _ = multi_tensor(gtab, mevals, S0, angles=[(0, 0), (60, 0)],
                            fractions=[50, 50], snr=SNR)
 
-    for sphere in [_sphere, default_sphere]:
+    for sphere in [_sphere, get_sphere('symmetric642')]:
 
         # test equality with/without multiprocessing
+        print("DEBUG -- 1")
```

Comment:
Please remove these prints. It seems overall in this PR you are trying to solve two problems.